proto/viam/app/v1/app.proto

```diff
@@ -843,11 +859,13 @@ enum FragmentErrorType {
   FRAGMENT_ERROR_TYPE_CYCLE_DETECTED = 4;
 }
 
+// Cached fragment usage statistics
```

Comment:
Do we want to add a comment here about how this will either return tag usages or revision usages?

Reply:
updated!